File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-python/gpt-j/sieve.py
--- 
+++ 
@@ -1,6 +1,6 @@
 def sieve(max):
     primes = []
     for n in range(2, max + 1):
-        if any(n % p > 0 for p in primes):
+        if all(n % i!= 0 for i in range(2, n)):
             primes.append(n)
     return primes

File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-java/incoder-1/sieve.java
--- 
+++ 
@@ -1,7 +1,14 @@
 public static ArrayList<Integer> sieve(Integer max) {
     ArrayList<Integer> primes = new ArrayList<Integer>();
     for (int n=2; n<max+1; n++) {
-        if (any(list_comp(n, primes))) {
+        boolean isPrime = true;
+        for (int i=2; i<n; i++) {
+            if (n % i == 0) {
+                isPrime = false;
+                break;
+            }
+        }
+        if (isPrime) {
             primes.add(n);
         }
     }
